src/renderer/components/wallet/Settings.style.tsx

```diff
@@ -188,3 +188,39 @@ export const StyledDeviceText = styled(Label)`
     margin-right: 10px;
   }
 `
+
+export const NetworkMenu = styled(A.Menu)`
+  background-color: ${palette('background', 1)};
+`
+
+export const ActionMenuItem = styled(A.Menu.Item)`
+  background-color: ${palette('background', 1)};
+  &:hover {
+    background-color: ${palette('background', 2)};
+  }
+`
+
+export const NetworkRow = styled(A.Row).attrs({
+  alignItems: 'center'
+})`
+  width: '150px';
```

Comment:
maybe its better to use `min-width` ?

Reply:
It seems we can't do this ^, it will expand the menu items in its width (see screenshot)

![Screenshot from 2020-09-01 18-23-49](https://user-images.githubusercontent.com/61792675/91880850-ed559780-ec80-11ea-8de5-e1921159cd26.png)


But we do need this:

![Screenshot from 2020-09-01 18-28-44](https://user-images.githubusercontent.com/61792675/91880889-fba3b380-ec80-11ea-8019-cf02f76c7bf9.png)

Let me know if you have another idea ...

Reply:
We can remove `NetworkRow` - its not used...